<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>BlankCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="Push_slider0">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:vertical{
border: 1px solid #AAA;
width: 50px;
background: #9AD9EB;
margin: 2px 0;
}
QSlider::handle:vertical{
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #B4B4B4, stop:1 #8f8f8f);
border: 1px solid gray;
height: 40px;
margin: -2px 0;
border-radius: 3px;
}</string>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Push_slider1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:vertical{
border: 1px solid #AAA;
width: 50px;
background: #9AD9EB;
margin: 2px 0;
}
QSlider::handle:vertical{
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #B4B4B4, stop:1 #8f8f8f);
border: 1px solid gray;
height: 40px;
margin: -2px 0;
border-radius: 3px;
}</string>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Push_slider2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:vertical{
border: 1px solid #AAA;
width: 50px;
background: #9AD9EB;
margin: 2px 0;
}
QSlider::handle:vertical{
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #B4B4B4, stop:1 #8f8f8f);
border: 1px solid gray;
height: 40px;
margin: -2px 0;
border-radius: 3px;
}</string>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Push_slider3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:vertical{
border: 1px solid #AAA;
width: 50px;
background: #9AD9EB;
margin: 2px 0;
}
QSlider::handle:vertical{
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #B4B4B4, stop:1 #8f8f8f);
border: 1px solid gray;
height: 40px;
margin: -2px 0;
border-radius: 3px;
}</string>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Push_slider4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:vertical{
border: 1px solid #AAA;
width: 50px;
background: #9AD9EB;
margin: 2px 0;
}
QSlider::handle:vertical{
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #B4B4B4, stop:1 #8f8f8f);
border: 1px solid gray;
height: 40px;
margin: -2px 0;
border-radius: 3px;
}</string>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Push_slider5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>170</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:vertical{
border: 1px solid #AAA;
width: 50px;
background: #9AD9EB;
margin: 2px 0;
}
QSlider::handle:vertical{
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #B4B4B4, stop:1 #8f8f8f);
border: 1px solid gray;
height: 40px;
margin: -2px 0;
border-radius: 3px;
}</string>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Push_slider6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:vertical{
border: 1px solid #AAA;
width: 50px;
background: #9AD9EB;
margin: 2px 0;
}
QSlider::handle:vertical{
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #B4B4B4, stop:1 #8f8f8f);
border: 1px solid gray;
height: 40px;
margin: -2px 0;
border-radius: 3px;
}</string>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Push_slider7">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:vertical{
border: 1px solid #AAA;
width: 50px;
background: #9AD9EB;
margin: 2px 0;
}
QSlider::handle:vertical{
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #B4B4B4, stop:1 #8f8f8f);
border: 1px solid gray;
height: 40px;
margin: -2px 0;
border-radius: 3px;
}</string>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Push_slider8">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>300</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:vertical{
border: 1px solid #AAA;
width: 50px;
background: #9AD9EB;
margin: 2px 0;
}
QSlider::handle:vertical{
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #B4B4B4, stop:1 #8f8f8f);
border: 1px solid gray;
height: 40px;
margin: -2px 0;
border-radius: 3px;
}</string>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>140</y>
      <width>311</width>
      <height>171</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 30pt &quot;Roboto&quot;;</string>
    </property>
    <property name="text">
     <string>PUSH _BUTTON</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
